packages/terra-badge/tests/browser/badge-spec.js

```diff
@@ -0,0 +1,32 @@
+/* global browser, it, viewport */
+describe('badge', () => {
+  const viewports = viewport('tiny', 'huge');
+
+  it('defaults', () => {
+    browser.url('/#/tests/badge-tests/default');
+    browser
```

Comment:
I think we can consolidate this all in a plugin so its no so verbose.

Reply:
EU did a good job of creating a command to do this in their repo https://github.com/ec-europa/ecl-toolkit/blob/master/packages/ecl-qa/wdio/assertions/matchReference.js